test/junit/scala/collection/ViewTest.scala

```diff
@@ -7,7 +7,7 @@ import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
 
 import language.postfixOps
-import scala.collection.mutable.ListBuffer
+import scala.collection.mutable.{ArrayBuffer, ListBuffer}
```

Comment:
Why did you decide to use `ArrayBuffer` if `ListBuffer` is already imported?  If you decide `ArrayBuffer` is better you should change the `ListBuffer` usages to `ArrayBuffer` to be consistent.